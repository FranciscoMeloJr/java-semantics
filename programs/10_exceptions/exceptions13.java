public class main {

  void main(string[] args) {
    try {
      throw 4;
    } catch (int e) {
      print(e," ");
    }
    print(42,"\n");
    print("Done!","\n");
  }
}

// 4 42
// Done!
