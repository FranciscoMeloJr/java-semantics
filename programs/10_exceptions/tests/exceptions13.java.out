4 42
Done!
